http/v1.md

```diff
@@ -0,0 +1,465 @@
+  *   makerAssetAddress [string] returns orders where the contract address for the maker asset matches the value specified
+  *   takerAssetAddress [string] returns orders where the contract address for the taker asset matches the value specified
+
+Order specific parameters:
+  *   exchangeAddress [string]: returns orders created for this exchange address
+  *   senderAddress [string]: returns orders with the specified senderAddress
+  *   makerAssetData [string]: returns orders with the specified makerAssetData
+  *   takerAssetData [string]: returns orders with the specified takerAssetData
+  *   traderAssetData [string]: returns orders where either makerAssetData or takerAssetData has the value specified
+  *   makerAddress [string]: returns orders with the specified makerAddress
+  *   takerAddress [string]: returns orders with the specified takerAddress
+  *   traderAddress [string]: returns orders where either makerAddress or takerAddress has the value specified
+  *   feeRecipientAddress [string]: returns orders where feeRecipientAddress is feeRecipient address
+
+All parameters are optional.
+
+If both makerAssetData and takerAssetData are specified, returned orders will be sorted by price determined by (takerTokenAmount/makerTokenAmount) in ascending order. By default, orders returned by this endpoint are unsorted.
+
+While there is some redundancy in supporting maker/takerAssetType, maker/takerAssetAddress, and maker/takerAssetData, they are actually all needed. For example, you cannot query "all ERC712 orders", or "all Cryptokitties orders", or "all ERC20 orders" with just maker/takerAssetData and need the other query params to do so.
+
+#### Response
+
+[See response schema](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/signed_orders_schema.ts#L1)
+
+
+```
+{
+    "total": 984,
+    "page": 1,
+    "per_page": 100,
+    "records": [
+        {
+          "order": {
+              "makerAddress": "0x9e56625509c2f60af937f23b7b532600390e8c8b",
+              "takerAddress": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
+              "feeRecipientAddress": "0xb046140686d052fff581f63f8136cce132e857da",
+              "senderAddress": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
+              "makerAssetAmount": "10000000000000000",
+              "takerAssetAmount": "20000000000000000",
+              "makerFee": "100000000000000",
+              "takerFee": "200000000000000",
+              "expirationTimeSeconds": "1532560590",
+              "salt": "1532559225",
+              "makerAssetData": "0xf47261b04c32345ced77393b3530b1eed0f346429d",
+              "takerAssetData": "0x0257179264389b814a946f3e92105513705ca6b990",
+              "exchangeAddress": "0x12459c951127e0c374ff9105dda097662a027093",
+              "signature": "0x012761a3ed31b43c8780e905a260a35faefcc527be7516aa11c0256729b5b351bc33"
+            },
+            "remainingFillableAmount": "5000000000000000"
```

Comment:
I had a thought here, could we make this field "orderInfo" instead of "remainingFillableAmount" order Info would simply be the output of doing a "getOrdersInfo" call to the exchange contract. Hydrating this information would require one rpc call

Reply:
This was discussed in https://github.com/0xProject/standard-relayer-api/issues/25 and it seems like people prefer to stick with the single variable.